pilot/pkg/features/pilot.go

```diff
@@ -400,6 +400,10 @@ var (
 			"Setting the timeout to 0 disables this behavior.",
 	).Get()
 
+	EnableTelemetry = env.RegisterBoolVar("PILOT_ENABLE_TELEMETRY", true,
```

Comment:
Maybe `ENABLE_TELEMETRY_LABEL`? `ENABLE_TELEMETRY` sound ambiguous to me.

Reply:
Ok. Makes sense. Changed.